src/ml/neural_net/image_augmentation.cpp

```diff
@@ -29,23 +28,25 @@ shared_float_array convert_to_shared_float_array(
     ann[offset + 5] = annotation.confidence;
   }
   shared_float_array data_to_augment =
-      shared_float_array::wrap(std::move(ann), {annotations_per_image.size() * 6});
+      shared_float_array::wrap(ann, {annotations_per_image.size(), 6});
```

Comment:
This should still use `std::move`, no?

Reply:
`shared_float_array::wrap` does it in its implementation.

Reply:
That's not how C++11 works. The implementation will now move a copy being made here. `std::move` must be used at every step along the way, otherwise a copy will occur